bot/exts/info/information.py

```diff
@@ -323,7 +323,15 @@ async def create_user_embed(self, ctx: Context, user: MemberOrUser, passed_as_me
         if is_mod_channel(ctx.channel):
             fields.append(await self.expanded_user_infraction_counts(user))
             fields.append(await self.user_nomination_counts(user))
-            fields.append(await self.user_alt_count(user))
+            try:
+                fields.append(await self.user_alt_count(user))
+            except ResponseCodeError as e:
+                if e.response.status == 404:
+                    fields.append(
+                        ("Associated accounts", "User not found in site database.")
+                    )
+                else:
+                    raise e
```

Comment:
Can we consider handling this in `self.user_alt_account`? 

Or,  we could document in `self.user_alt_account` that a `ResponseCodeError` will be raised if the user is not found in the database (have I been using too much Java?) .

Reply:
Changed to this way in latest force push!